{
    "CVE-2019-10856": {
        "timescope": [
            ["18h23m03s", "18h25m00s"]
        ],
        "tagged_library": [ 
            "pypi__split__jupyter(https://pypi.org/project/jupyter/)"
        ],
        "reason":"https://github.com/jupyter/notebook/compare/16cf97c...b8e30ea"
    },
    "CVE-2019-10795": {
        "timescope": [
            ["18h26m05s", "18h28m05s"]
        ],
        "tagged_library": [ 
            "npm__split__undefsafe(https://github.com/remy/undefsafe)"
        ],
        "reason":"https://security.snyk.io/vuln/SNYK-JS-UNDEFSAFE-548940,https://security.snyk.io/package/npm/undefsafe"
    },
    "CVE-2021-29469": {
        "timescope": [
            ["18h28m20s", "18h31m04s"]
        ],
        "tagged_library": [ 
            "npm__split__node-redis(https://redis.js.org/)"
        ],
        "reason":"https://github.com/redis/node-redis/security/advisories/GHSA-35q2-47q7-3pc3"
    },
    "CVE-2021-32641": {
        "timescope": [
            ["18h31m31s", "18h35m06s"]
        ],
        "tagged_library": [ 
            "npm__split__auth0-lock(https://auth0.com/docs/libraries/lock)"
        ],
        "reason":"https://github.com/auth0/lock/security/advisories/GHSA-jr3j-whm4-9wwm"
    },
    "CVE-2017-16618": {
        "timescope": [
            ["18h35m21s", "18h38m16s"]
        ],
        "tagged_library": [ 
            "pypi__split__owlmixin(https://pypi.org/project/owlmixin/)"
        ],
        "reason":"https://github.com/tadashi-aikawa/owlmixin/blob/master/owlmixin/util.py"
    },
    "CVE-2022-30284": {
        "timescope": [
            ["18h38m40s", "18h41m10s"]
        ],
        "tagged_library": [ 
            "pypi__split__python-libnmap(https://libnmap.readthedocs.io/en/latest/)"
        ],
        "reason":"https://github.com/savon-noir/python-libnmap/releases"
    }
}


